### PR TITLE
Adapt sumaform to make use of the new 'evil-minions'

### PR DESCRIPTION
**Pull request by chiaradiamarcelo**
Adapt sumaform to make use of the new 'evil-minions'.
This PR includes:
- remove the old support for old evil-minions, in libvirt and openstack 
- make use of the new evil-minions in libvirt
- adapt the PTS for making use of the new evil minions in 'libvirt'
- adapt the PTS for making use of the new evil minions in 'openstack'

### CODE REVIEW

**Comment by moio** [comment]
Please check Travis CI errors

File: README_ADVANCED.md [hunk 1]
```diff
@@ -668,28 +658,7 @@ module "minion" {
...
-```hcl
-module "evil-minions" {
-  source = "./modules/libvirt/evil_minions"
-  base_configuration = "${module.base.configuration}"
-
-  name = "evil-minions"
-  server_configuration = "${module.suma31pg.configuration}"
-  dump_file = "./minion-dump.mp"
-  evil_minion_count = 50
+  evil_minion_count = true
```
Comment (aaannz) [inline README_ADVANCED.md]: This should be number instead of boolean, right?

File: salt/suse_manager_server/pts/run-pts.py [hunk 2]
```diff
@@ -79,25 +140,24 @@ def run_locust_http_load(clients_count):
...
 
+if "locust" in enabled_phases:
+    for users in range(50, 450, 25):
+        run_locust_http_load(users)
+
+if "patching" in enabled_phases:
+    set_up()
+    patch_all_systems()
+
 if "fio" in enabled_phases:
```
Comment (moio) [inline salt/suse_manager_server/pts/run-pts.py]: fio was supposed to run before real testing phases, can we move it over?

File: modules/libvirt/minion/variables.tf [hunk 3]
```diff
@@ -28,9 +28,14 @@ variable "auto_connect_to_master" {
...
-variable "evil_minions_dump" {
-  description = "whether this minion should create a dump file to be used for evil-minions"
-  default = false
+variable "evil_minion_count"  {
+  description = "change to a number to use the evil-minions load generator, see README_ADVANCED.md"
+  default = 0
+}
+
+variable "evil_minion_slowdown_factor"  {
+  description = "slowdown factor for evil-minions, see README_ADVANCED.md"
```
Comment (moio) [inline modules/libvirt/minion/variables.tf]: Please either document in README_ADVANCED or point to evil-minions documentation instead

File: salt/suse_manager_server/pts/run-pts.py (hunk 2 above)
Comment (chiaradiamarcelo) [reply salt/suse_manager_server/pts/run-pts.py]: sure

**Pull request merged**